README.md

```diff
@@ -104,7 +104,7 @@ For more information, please refer to the developer documentation of these servi
 * [Okta](https://developer.okta.com/authentication-guide/implementing-authentication/auth-code)
 * [BitBucket](https://confluence.atlassian.com/bitbucket/oauth-on-bitbucket-cloud-238027431.html)
 
-Further information on the OAuth endpoints can also be found [here](https://tools.ietf.org/html/rfc6749#page-21).
+Further information on the OAuth endpoints, see the Internet Engineering Task Force (IETF) RFC 6749, [The OAuth 2.0 Authorization Framework](https://tools.ietf.org/html/rfc6749), page 21.
```

Comment:
Should it be "For further information on the OAuth endpoints..." ?

Reply:
Changed.